runtime/core/exec_aten/util/scalar_type_util.h

```diff
@@ -1204,26 +1201,17 @@ struct promote_types {
   ET_SWITCH_REAL_TYPES_AND3(                                              \
       Half, Bool, BFloat16, TYPE, CONTEXT, NAME, CTYPE_ALIAS, __VA_ARGS__)
 
+#define ET_SWITCH_ALLREALB_TYPES(TYPE, CONTEXT, NAME, CTYPE_ALIAS, ...) \
+  ET_INTERNAL_SWITCH(                                               \
+      TYPE,                                                         \
+      CONTEXT,                                                      \
+      NAME,                                                         \
+      ET_INTERNAL_SWITCH_CASE_ALLREALB_TYPES(CTYPE_ALIAS, __VA_ARGS__))
+
+// Deprecated: Use ET_SWITCH_ALLREALB_TYPES instead.
 #define ET_SWITCH_REALHBBF16_AND_UINT_TYPES(                  \
     TYPE, CONTEXT, NAME, CTYPE_ALIAS, ...)                    \
-  ET_INTERNAL_SWITCH(                                         \
-      TYPE,                                                   \
-      CONTEXT,                                                \
-      NAME,                                                   \
-      ET_INTERNAL_SWITCH_CASE_REAL_TYPES_AND3(                \
-          Half, Bool, BFloat16, CTYPE_ALIAS, __VA_ARGS__)     \
-          ET_INTERNAL_SWITCH_CASE(                            \
-              ::executorch::aten::ScalarType::UInt16,         \
-              CTYPE_ALIAS,                                    \
-              __VA_ARGS__)                                    \
-              ET_INTERNAL_SWITCH_CASE(                        \
-                  ::executorch::aten::ScalarType::UInt32,     \
-                  CTYPE_ALIAS,                                \
-                  __VA_ARGS__)                                \
-                  ET_INTERNAL_SWITCH_CASE(                    \
-                      ::executorch::aten::ScalarType::UInt64, \
-                      CTYPE_ALIAS,                            \
-                      __VA_ARGS__))
+  ET_SWITCH_ALLREALB_TYPES(TYPE, CONTEXT, NAME, CTYPE_ALIAS, __VA_ARGS__)
```

Comment:
this is more easily accomplished as `#define ET_SWITCH_REALHBBF16_AND_UINT_TYPES ET_SWITCH_ALLREALB_TYPES` (and similarly below)